thunder/executors/nvfuserex_impl.py

```diff
@@ -3193,3 +3193,70 @@ def argsort_transform(
 assert NVFUSER_SUPPORTS_OPTIONS, (
     f"Installed version of nvFuser {nvfuser_version()} is not supported, please upgrade to 0.2.23 or later."
 )
+
+
+def _scaled_mm_check(
+    a: TensorLike,
+    b: TensorLike,
+    scale_a: TensorLike,
+    scale_b: TensorLike,
+    bias: TensorLike | None = None,
+    scale_result: TensorLike | None = None,
+    out_dtype: dtypes.dtype | None = None,
+    use_fast_accum: bool = False,
+) -> bool:
+    return nvfuser_version() >= LooseVersion("0.2.27")
+
+
+# FIXME: Somehow the expected fd.ops does not seem to appear in the trace
+def _scaled_mm(
+    a: TensorLike,
+    b: TensorLike,
+    scale_a: TensorLike,
+    scale_b: TensorLike,
+    bias: TensorLike | None = None,
+    scale_result: TensorLike | None = None,
+    out_dtype: dtypes.dtype | None = None,
+    use_fast_accum: bool = False,
+    *,
+    fd: FusionDefinition,
+    lc_to_nv_map: dict,
+):
+    mat1 = getnv(a, fd, lc_to_nv_map)
+    mat2 = getnv(b, fd, lc_to_nv_map)
+    scale1 = getnv(scale_a, fd, lc_to_nv_map)
+    scale2 = getnv(scale_b, fd, lc_to_nv_map)
+    if bias is not None:
+        nv_bias = getnv(bias, fd, lc_to_nv_map)
+    else:
+        nv_bias = None
+    if out_dtype is not None:
+        out_dtype = dtypes.to_dtype(out_dtype)
+    if out_dtype is not None:
+        dtype = lcdtype_to_nvdtype(out_dtype)
+    else:
+        dtype = None
+    # TODO: figure out how to use `scale_result` in `fd.ops.scaled_mm`
```

Comment:
scale_result is just alph.

Reply:
output_block_scale_size and output_block_scale_dtype are meant for outputting quantized tensor.

Reply:
Note: @crcrpar I'm not sure how *stable* this API would be.

I don't yet know what the cutlass API would look like. I vaguely remember reading some other docs saying that global scaling factor on the output might need to be provided as input to grouped_mm.